docs/user/lib_mapper/scrapli.md

```diff
@@ -1,5 +1,6 @@
 | NORMALIZED | | SCRAPLI |
 | ---------- | -- | ------ |
+| aruba_aoscx | → | aruba_aoscx |
```

Comment:
Sorry, there is confusion. This is an artifact, not the source. You have to update the actual mappers and then generate this.